pkg/runtime/kubernetes/utils.go

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubernetes
+
+import (
+	"fmt"
+
+	"net"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/sealerio/sealer/common"
+	"github.com/sealerio/sealer/utils/exec"
+	osi "github.com/sealerio/sealer/utils/os"
+	"github.com/sealerio/sealer/utils/ssh"
+
+	"github.com/pkg/errors"
+)
+
+//kubeVersion is a []string that we use to normalize version string.
+type kubeVersion []string
```

Comment:
`type KubeVersion string` here.
And you can init a KubeVersion like KubeVersion("v1.21.4").
And I think this struct could be put in a single file?